src/App.js

```diff
@@ -112,6 +113,7 @@ class Root extends React.Component {
               <NavigationRoute exact path="/transactions" component={TransactionList} />
               <NavigationRoute exact path="/tokens" component={TokenList} />
               <NavigationRoute exact path="/token_balances" component={TokenBalancesList} />
+              <NavigationRoute exact path="/timeseries" component={TimeSeriesDashboard} />
```

Comment:
I know that the dashboard is meant to display timeseries charts, but should we show `Time Series` and `/timeseries` to the user? Is there not a more user friendly name for this feature?

Reply:
I will remove it and put on the `Statistics` page